doc/model.rst

```diff
@@ -0,0 +1,683 @@
+    exp = Experiment(name="getting-started", launcher="local")
+
+Models require run settings objects which are initialized through an Experiment instance.
+Use the experiment instance, `exp`, to access the helper method ``Experiment.create_run_settings()``
+to initialize a ``RunSettings`` object. In this example, we use the Python
+executable to execute the application script, "script.py":
+
+.. code-block:: python
+
+    settings = exp.create_run_settings(exe="python", exe_args="script.py")
+
+We now have an instance of run settings name `settings`. Using the instance, instruct SmartSim
+to encapsulate the simulation as a Model through ``Experiment.create_model()``:
+
+.. code-block:: python
+
+    model = exp.create_model(name="example-model", run_settings=settings)
+
+Remember that all entities are launched, monitored and stopped by the Experiment instance.
+Specify the model instance, `model`, to ``Experiment.start()`` to start the standard model:
+
+.. code-block:: python
+
+    exp.start(model)
+
+When the Experiment Python driver script is executed, two files from the standard model will be created
+in the Experiment working directory:
+
+1. `model.out` : this file will hold outputs produced by the Model workload
+2. `model.err` : will hold any errors that occurred during Model execution
+
+-----------------
+A Colocated Model
+-----------------
+A colocated model is when both the application and the
+database share compute resource. To create a colocated model,
+users first initialize a Model instance with the ``Experiment.create_model()`` function.
+A user must then colocate the model with the Model API function ``Model.colocate_db()``.
+This instructs SmartSim to launch an Orchestrator on the same compute
+nodes as the model when the model instance is deployed via ``Experiment.start()``.
+
+There are **three** different Model API helper functions to colocate a Model:
+
+- ``Model.colocate_db()``: An alias for `Model.colocate_db_tcp()`.
+- ``Model.colocate_db_tcp()``: Colocate an Orchestrator instance with this Model over TCP/IP.
+- ``Model.colocate_db_uds()``: Colocate an Orchestrator instance with this Model over UDS.
+
+Each function will initialize an unsharded database to the Model instance. Only the partnered Model
+may communicate with the database via a SmartRedis client by using the loopback TCP interface or
+Unix Domain sockets. Extra parameters for the database can be passed into the helper functions
+via kwargs.
+
+.. code-block:: python
+
+    example_kwargs = {
+        "maxclients": 100000,
+        "threads_per_queue": 1,
+        "inter_op_threads": 1,
+        "intra_op_threads": 1
+    }
+
+For a walkthrough of how to colocate a Model, navigate to the :ref:`Colocated Orchestrator<link>` for
+instructions.
+
+=====
+Files
+=====
+--------
+Overview
+--------
+The ``Model.attach_generator_files()`` function enables users to write model
+parameters to a file or attachment a file of parameters with values to the
```

Comment:
awkward wording: "write model parameters to a file _or attachment a file_ of..."

- Consider "enables the use of external files with a model.\n"

- Consider avoiding the repetition of to_copy, to_symlink, to_configure in rapid succession (IOW, "users can specify whether..." could be deleted in favor of follow-up paragraph detailing usage)

- Consider using a [note](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html#notes) for "it's worth noting..." blurb